types/params.go

```diff
@@ -12,6 +12,12 @@ const (
 
 	// BlockPartSizeBytes is the size of one block part.
 	BlockPartSizeBytes = 65536 // 64kB
+
+	// MaxStateSizeBytes is the maximum permitted size of the blocks.
+	// TODO: reduce this by segmentation
+	MaxStateSizeBytes = 1048576000 // 1000MB
+
+	MonitorWindowInSeconds = 40
```

Comment:
doesn't seem to below here. These are hard consensus params, but this is more like a reactor setting.